tools/perf/components/browser_type.py

```diff
@@ -19,11 +19,57 @@
 from zipfile import ZipFile
 from distutils.dir_util import copy_tree
 
-from components import path_util
+import components.path_util as path_util
 from components.perf_test_utils import GetProcessOutput
 
 
-def _GetBraveDownloadUrl(tag: str, binary: str) -> str:
+class _BaseVersion:
+  _version: List[int]
+
+  def equals(self, a: _BaseVersion) -> bool:
+    return self._version == a.version()
+
+  def less(self, a: _BaseVersion) -> bool:
+    return self._version < a.version()
+
+  def version(self) -> List[int]:
+    return self._version
+
+
+class BraveVersion(_BaseVersion):
+
+  def __init__(self, tag: str) -> None:
+    super().__init__()
+    assert re.match(r'v\d+\.\d+\.\d+', tag)
+    self._version = list(map(int, tag[1:].split('.')))
+
+  def __str__(self) -> str:
+    return 'v' + '.'.join(map(str, self._version))
+
+  def to_chromium_version(self) -> 'ChromiumVersion':
+    return _GetChromiumVersion(self)
```

Comment:
is `git show` quick enough? how many calls to this method? maybe we should have a cached map `brave_version->chromium.major`? or at least cache what `_GetChromiumVersion` returns if we only work with a single `BraveVersion` during the run.

Reply:
We call `git something` a few times per each test binary. It adds about 10sec to the test binary time. In total we have about 30min, so it doesn't look like a place to optimize now. 

After an integration with CI some git operations will be removed, so we can think about caching that time.